app/app.go

```diff
@@ -350,7 +370,7 @@ func NewApp(
 		appCodec,
 		keys[authtypes.StoreKey],
 		authSubspace,
-		authtypes.ProtoBaseAccount,
+		ethermint.ProtoAccount,
```

Comment:
this changes all BaseAccounts to an EthAccount, which is unlikely what we will do. However, keeping it this for now to simplify dev before we figure out how to properly handle this.